lib/mato/html_filters/mention_link.rb

```diff
@@ -37,7 +37,7 @@ def call(doc)
           next if text_node.content == candidate_html
 
           candidate_fragment = text_node.replace(candidate_html)
```

Comment:
in case of `text_node`'s text is `@valid\<span\>\<\/span\>`, `candidate_html` is `<span class='mention-candidate'>@valid</span><span></span>`.
So an error occur at L41. (`<span></span>` doesn't have `child.content`)

Reply:
But in the case of `@valid\<span\ class\=\'mention-candidate\'>\<\/span\>`, I think that an error will occur, maybe.
However, it doesn't correspond because it is an edge case.

Reply:
I guess the main cause is it evaluates text node content as DOM. So the best solution is avoiding to evaluate the content.

But I think this pull request is acceptable if we need to fix the bug ASAP.

Reply:
How about `'@valid\<span class="mention-candidate"\>\<\/span\>'` ?

Reply:
yes.